src/app/header/header.component.ts

```diff
@@ -1,12 +1,29 @@
-import { Component, OnInit } from '@angular/core';
+import {
+  Component, EventEmitter, Output,
+} from '@angular/core';
 
 @Component({
   selector: 'app-header',
   templateUrl: './header.component.html',
   styleUrls: ['./header.component.scss'],
 })
-class HeaderComponent implements OnInit {
-  ngOnInit(): void {
+class HeaderComponent {
+  searchKeyword = '';
```

Comment:
same thing for the types - define in all places for variable and functions to have it more consistancy across app :)